modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2099,6 +2102,79 @@ export class Wallet implements IWallet {
     return this.sendMany(sendManyOptions);
   }
 
+  async sendNft(sendOptions: SendNFTOptions, sendNftOptions: NFTTransferOptions): Promise<SendNFTResult> {
+    const nftCollections = await this.getNftBalances();
+    const { tokenContractAddress, recipientAddress, type } = sendNftOptions;
+
+    const nftBalance = nftCollections.find((c) => c.metadata.tokenContractAddress === tokenContractAddress);
+    if (!nftBalance) {
+      throw new Error(`Collection not found for token contract ${tokenContractAddress}`);
+    }
+
+    if (!this.baseCoin.isValidAddress(recipientAddress)) {
+      throw new Error(`Invalid recipient address ${recipientAddress}`);
+    }
+    const baseAddress = this.coinSpecific()?.baseAddress;
+    if (!baseAddress) {
+      throw new Error('Missing base address for wallet');
+    }
+
+    if (nftBalance.type !== type) {
+      throw new Error(`Specified NFT type ${type} does not match collection type ${nftBalance.type}`);
+    }
+
+    switch (sendNftOptions.type) {
+      case 'ERC721': {
+        if (!nftBalance.collections[sendNftOptions.tokenId]) {
+          throw new Error(
+            `Token ${sendNftOptions.tokenId} not found in collection ${tokenContractAddress} or does not have a spendable balance`
+          );
+        }
+
+        const data = this.baseCoin.buildNftTransferData({ ...sendNftOptions, fromAddress: baseAddress });
+        return this.sendMany({
+          ...sendOptions,
+          recipients: [
+            {
+              address: sendNftOptions.tokenContractAddress,
+              amount: '0',
+              data: data,
+            },
+          ],
+        });
+      }
+      case 'ERC1155': {
+        const entries = sendNftOptions.entries;
+        for (const entry of entries) {
+          if (!nftBalance[entry.tokenId]) {
+            throw new Error(
+              `Token ${entry.tokenId} not found in collection ${sendNftOptions.tokenContractAddress} or does not have a spendable balance`
+            );
+          }
+          if (nftBalance[entry.tokenId] < entry.amount) {
+            throw new Error(
+              `Amount ${entry.amount} exceeds spendable balance of ${nftBalance[entry.tokenId]} for token ${
+                entry.tokenId
+              }`
+            );
+          }
+        }
+
+        const data = this.baseCoin.buildNftTransferData({ ...sendNftOptions, fromAddress: baseAddress });
+        return this.sendMany({
+          ...sendOptions,
+          recipients: [
+            {
+              address: sendNftOptions.tokenContractAddress,
+              amount: '0',
```

Comment:
Is there a case where we'd have any fees and it doesnt make sense for `amount` to be 0?

Reply:
Don't think so, the documentation https://developers.bitgo.com/guides/transact/nfts says to always set the amount to 0.